data/Kubiak_2009/metadata.yml

```diff
@@ -0,0 +1,2161 @@
+    to the fires was drawn upon to help locate particular species that might otherwise
+    have been overlooked. (Some new 'discoveries' of species, previously unrecorded
+    by the author, were made following the fires). The main references used for plant
+    identification were Harden (1990-1993, 2002), Harden & Murray (2000), Carolin
+    & Tindale (1994), Fairley & Moore (1989) and Robinson (1991).
+  original_file: Data were transcribed; see original transcribed data at 'data/Kubiak_2009/raw/wide_data.csv'.
+  notes: Data were transcribed.
+locations:
+  Lane Cove River catchment, Sydney:
+    latitude (deg): -33.75
+    longitude (deg): 151.08
+    description: Clarke & Benson (1987) mapped and described 15 vegetation types,
+      including mangrove forest, saltmarsh, tall forest, open-forest, woodland, shrubland
+      and riparian shrubland for Lane Cove National Park (then known as Lane Cove
+      River State Recreation Area). Most of the surviving bushland in the Lane Cove
+      River area is sclerophyllous and occurs on sandstone geology, described in broad
+      terms by Keith (2004) as the Sydney Coastal Dry Sclerophyll Forests.
+    locality: northern Sydney metropolitan area
+    geology (parent material): sandstone
+    fire history: Generally, the study areas have experienced bushfires quite frequently
+      in the recent past. However, there may be some patches of bushland within these
+      catchments that escape being burnt for fairly long periods of time.
+  Narrabeen Lagoon catchment, Sydney:
+    latitude (deg): -33.71
+    longitude (deg): 151.24
+    description: The floristic diversity of the Narrabeen Lagoon catchment area is
+      comparable to that of the Lane Cove River area. Sheringham & Sanders (1993)
+      mapped and described 21 vegetation types in the eastern section of Garigal National
+      Park (centred on Deep Creek) including areas of openforest, woodland, heathland,
+      wetland, swamp and closedforest. Heathland is much more common at the Narrabeen
+      Lagoon area than at Lane Cove River (see Benson & Howell 1990, 1994).
+    locality: northern Sydney metropolitan area
+    geology (parent material): .na
+    fire history: Generally, the study areas have experienced bushfires quite frequently
+      in the recent past. However, there may be some patches of bushland within these
+      catchments that escape being burnt for fairly long periods of time.
+  Lane Cove River and/or Narrabeen Lagoon catchment areas, Sydney:
+    latitude (deg): -33.75
+    longitude (deg): 151.15
+    description: .na
+    locality: northern Sydney metropolitan area
+    geology (parent material): .na
+    fire history: Generally, the study areas have experienced bushfires quite frequently
+      in the recent past. However, there may be some patches of bushland within these
+      catchments that escape being burnt for fairly long periods of time.
+contexts:
```

Comment:
I've had to add an additional context "replicate observations" to capture the multiple values for a single taxon for the same trait for the same population for the same context. If you run 
```
austraits$traits %>% filter(dataset_id == current_study) %>% 
  select(dataset_id, trait_name, value, observation_id, source_id, taxon_name, population_id, individual_id, temporal_id, method_id, entity_context_id, value_type, original_name) %>%
  pivot_wider(names_from = trait_name, values_from = value, values_fn = length) %>% View()
```
All the values should be 1's, indicating that there is a single value for this cluster of variables. Usually you just group by taxon, location, trait and add row numbers.  In this case, because there were lots of trait names that were mapped to `time from fire to flowering` I had to cluster those traits together first.